### PR TITLE
Drawio in vpc model

### PR DESCRIPTION
**Pull request by haim-kermany**
integrated drawio into vpc model
updating tests in ibmvpc

### CODE REVIEW

File: pkg/vpcmodel/abstractVPC.go [hunk 1]
```diff
@@ -21,6 +21,9 @@ type NamedResourceIntf interface {
 	UID() string
 	Name() string
 }
+type ZonalNamedResourceIntf interface {
+	ZoneName() string
+}
```
Comment (adisos) [inline pkg/vpcmodel/abstractVPC.go:26]: might be simpler to add `ZoneName()` to the `NamedResourceIntf` ?

**Review by adisos — COMMENTED** [review]
looks good, just added one comment

File: pkg/vpcmodel/abstractVPC.go (hunk 1 above)
Comment (haim-kermany) [reply pkg/vpcmodel/abstractVPC.go:26]: not all NamedResourceIntf  has zone, only the one that inherit zonalNamedResource

Comment (haim-kermany) [reply pkg/vpcmodel/abstractVPC.go]: fixed, removed ZonalNamedResourceIntf  and ZonalNamedResource

**Review by adisos — APPROVED** [review]

**Pull request merged**